### PR TITLE
Add Roc.yml

### PR DESCRIPTION
**Pull request by gavv**
Add Roc project, a toolkit for real-time audio streaming over the network:
https://github.com/roc-project/roc/

### CODE REVIEW

File: _data/projects/Roc.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://roc-project.github.io/
+tags:
+- C++
+- Audio
+- DSP
+- streaming
+- networking
+upforgrabs:
+  name: help wanted
+  link: https://github.com/roc-project/roc/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (ritwik12) [inline _data/projects/Roc.yml]: @gavv Please change this to https://github.com/roc-project/roc/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Roc.yml (hunk 1 above)
Comment (gavv) [reply _data/projects/Roc.yml]: Done.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@gavv Thanks!!

**Comment by gavv** [comment]
@ritwik12 thanks too, and I guess the tag "needs work" can be removed now.

**Review by shiftkey — APPROVED** [review]

**Pull request merged**